### PR TITLE
housekeeping changes for SAPHanaSrCostOptMemConfig.py

### PR DESCRIPTION
**Pull request by fmherschel**
SAPHanaSrCostOptMemConfig.py:
- corrected some comments in the code about method-parameters
- removed unneeded (empty) methods - only keep pre/post takeover and init.
- added global_allocation_init parameter example for global.ini in the inline example
- added/corrected TODOs

### CODE REVIEW

**Review by angelabriel — CHANGES_REQUESTED** [review]
Except of the 2 questions regarding the 'global.ini' file it looks really good to me. Especially I highly appreciate the slimming down of the hook code.

File: srHook/global.ini [hunk 1]
```diff
@@ -1,15 +1,10 @@
...
-[ha_dr_provider_SAPHanaSR]
-provider = SAPHanaSR
-path = /usr/share/SAPHanaSR
-execution_order = 1
-
 [ha_dr_provider_SAPHanaSrCostOptMemConfig]
 provider = SAPHanaSrCostOptMemConfig
 path = /usr/share/SAPHanaSR
 userKey = costoptkey
 execution_order = 2
```
[the remark this answers is not in the document]
Comment (fmherschel) [reply srHook/global.ini:6]: But also for CostOpt the customer always needs to integrate the SAPHanaSR hook. So execution_order = 2 is ok for the second hook to be added in my opinion.
We could either build a global.ini example per Use-Case or per Hook.

File: srHook/global.ini [hunk 2]
```diff
@@ -1,15 +1,10 @@
 
-[ha_dr_provider_SAPHanaSR]
-provider = SAPHanaSR
-path = /usr/share/SAPHanaSR
-execution_order = 1
-
```
[the remark this answers is not in the document]
Comment (fmherschel) [reply srHook/global.ini:6]: oh oh I mixed them  up - correction follows :)

File: srHook/global.ini (hunk 1 above)
[the remark this answers is not in the document]
Comment (fmherschel) [reply srHook/global.ini]: I would elect the per-hook global.ini examples.

**Review by angelabriel — APPROVED** [review]
Many thanks for the explanation and the change. Looks pretty good for me.

**Pull request merged**